src/build.py

```diff
@@ -1130,99 +1134,112 @@ def DebianPackage():
 
 
 def CompileLLVMTorture(extension, outdir):
-  name = 'Compile LLVM Torture (%s)' % extension
-  buildbot.Step(name)
-  c = Executable(os.path.join(INSTALL_BIN, 'clang'))
-  cxx = Executable(os.path.join(INSTALL_BIN, 'clang++'))
-  Remove(outdir)
-  Mkdir(outdir)
-  unexpected_result_count = compile_torture_tests.run(
-      c=c, cxx=cxx, testsuite=GCC_TEST_DIR,
-      sysroot_dir=INSTALL_SYSROOT,
-      fails=LLVM_KNOWN_TORTURE_FAILURES,
-      out=outdir,
-      config='wasm-' + extension)
-  UploadArchive('torture-' + extension, Archive(outdir))
-  if 0 != unexpected_result_count:
-    buildbot.Fail()
+  for opt in BARE_TEST_OPT_FLAGS:
```

Comment:
I think it might be a little more clear/readable if CompileLLVMTorture (and Link/Assemble/Execute) just did a single compile (and took an argument for the opt level), and we put the `for opt in BARE_TEST_OPT_FLAGS` out in the caller. 
That way we'd get one test run per call, and that would be consistent with the way we call it once for lld and once for s2wasm. And if we later decide we don't want the full cross-product of optlevel x lld it would be easier to change.

Reply:
Done